index.js

```diff
@@ -10,13 +10,14 @@ const imageExtensions = ['gif', 'png', 'svg', 'ico', 'jpg', 'jpeg']
 const musicExtensions = ['mp3', 'wav', 'aiff']
 const videoExtensions = ['avi', 'mpg', 'mpeg', 'mp4']
 const pdfExtensions = ['pdf']
-
 const defaultReqOptions = {
   timeout: 6000,
   maxRedirects: 30,
   userAgent: 'Schenkerianbot/1.0 (+https://github.com/mix/schenkerian)'
 }
 
+const RE_SPACES = /\s+/g
```

Comment:
Note that  this will also  match a space+line-break -- is that OK for the output of  this module?

Reply:
Yes. Line breaks are generally irrelevant for grabbing content.